setup.py

```diff
@@ -119,7 +118,7 @@ def download_url(version):
 
 
 setup(name="mb-rngpy",
-      python_requires="~=2.7",
+      python_requires="~=3.5",
```

Comment:
Why is this particular version of Python 3.x required? (Documenting in commit message would be nice.)

Reply:
updated

Reply:
doesn’t seem so

Reply:
check commit message.generic py version